cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -546,7 +535,9 @@ private void addProjectLocksHandlers(String path, RouteRole[] requiredRoles) {
         String pathWithoutResource = path.replace(getResourceId(path), "");
 
         get(path, new ProjectLockGetOne(metrics));
+        addCacheControl(path, 5, TimeUnit.MINUTES);
```

Comment:
I'm not sold on caching project locks. Seems like you always want the current state for this workflow